pkg/webhook/workspace/handler/handler.go

```diff
@@ -0,0 +1,35 @@
+package handler
+
+import (
+	"encoding/json"
+	"k8s.io/apimachinery/pkg/runtime"
+	"net/http"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+type WebhookHandler struct {
+	Client  client.Client
```

Comment:
Is this `client.Client` used currently?

Reply:
Good catch! Now it's used, I don't create client in exec.go but reuse this one.